xbmc/commons/ilog.h

```diff
@@ -53,6 +53,7 @@
 #define LOGCEC        (1 << (LOGMASKBIT + 9))
 #define LOGVIDEO      (1 << (LOGMASKBIT + 10))
 #define LOGWEBSERVER  (1 << (LOGMASKBIT + 11))
+#define LOGTIMING     (1 << (LOGMASKBIT + 12))
```

Comment:
I specifically mean audio and video decode/rendering timing, so maybe `LOGAVTIMING` is better? Or is it still too generic?